python-backend/app/api/parties/party/models/party_address.py

```diff
@@ -0,0 +1,17 @@
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.schema import FetchedValue
+
+from ....utils.models_mixins import Base
+from app.extensions import db
+
+class PartyAddress(Base):
```

Comment:
xref suffix just for consistency?

Reply:
I didn't use an `Xref` suffix in the Variances class, so it wouldn't have been consistent with that. I, personally, think that this is plenty clear (party + address = PartyAddress) and the `Xref` is a bit of info about the db leaking into the application layer that probably isn't needed.

But I'm also fine with renaming it if we think it helps.

Reply:
lots of the other xref tables have xref suffix on the class name. I think it should be as close to the table name as possible.